RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stdout maxBuffer length exceeded
    at new NodeError (node:internal/errors:278:15)
    at Socket.onChildStdout (node:child_process:409:14)
    at Socket.emit (node:events:376:20)
    at addChunk (node:internal/streams/readable:311:12)
    at readableAddChunk (node:internal/streams/readable:282:11)
    at Socket.Readable.push (node:internal/streams/readable:225:10)
    at Pipe.onStreamRead (node:internal/stream_base_commons:192:23)TypeError: fs.createWriteStream is not a function
    at imageGetHandler (file:///Users/Diatta/WebstormProjects/photagallery/src/api/image/image_routes.mjs:69:28)
    at dispatch (/Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:42:32)
    at validateParams (file:///Users/Diatta/WebstormProjects/photagallery/src/utils/koa-utilities.mjs:15:15)
    at dispatch (/Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:42:32)
    at /Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:42:32)
    at /Users/Diatta/WebstormProjects/photagallery/node_modules/@koa/router/lib/router.js:372:16
    at dispatch (/Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:42:32)
    at /Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/Diatta/WebstormProjects/photagallery/node_modules/@koa/router/lib/router.js:377:31)TypeError: stream.pipe is not a function
    at imageGetHandler (file:///Users/Diatta/WebstormProjects/photagallery/src/api/image/image_routes.mjs:71:12)
    at dispatch (/Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:42:32)
    at validateParams (file:///Users/Diatta/WebstormProjects/photagallery/src/utils/koa-utilities.mjs:15:15)
    at dispatch (/Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:42:32)
    at /Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:42:32)
    at /Users/Diatta/WebstormProjects/photagallery/node_modules/@koa/router/lib/router.js:372:16
    at dispatch (/Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:42:32)
    at /Users/Diatta/WebstormProjects/photagallery/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/Diatta/WebstormProjects/photagallery/node_modules/@koa/router/lib/router.js:377:31)Error: This socket has been ended by the other party
    at PromiseConnection.query (/Users/Diatta/WebstormProjects/photagallery/node_modules/mysql2/promise.js:92:22)
    at imageIdExists (file:///Users/Diatta/WebstormProjects/photagallery/src/utils/sql-utils.mjs:15:30)
    at addImageMetadata (file:///Users/Diatta/WebstormProjects/photagallery/src/utils/sql-utils.mjs:21:17)
    at imagePostHandler (file:///Users/Diatta/WebstormProjects/photagallery/src/api/image/image_routes.mjs:40:26)
    at async file:///Users/Diatta/WebstormProjects/photagallery/src/server.mjs:28:9
    at async serve (/Users/Diatta/WebstormProjects/photagallery/node_modules/koa-static/index.js:53:9)Error: Command failed: aws --endpoint-url http://localhost:8333 s3 cp /var/folders/r0/ktfq_wg90b18mrfb034n78wr0000gn/T/hqOvUgdGUbqiDyYs_low s3://waiting_on_bug_bucket
upload failed: ../../../../var/folders/r0/ktfq_wg90b18mrfb034n78wr0000gn/T/hqOvUgdGUbqiDyYs_low to s3://waiting_on_bug_bucket/hqOvUgdGUbqiDyYs_low Could not connect to the endpoint URL: "http://localhost:8333/waiting_on_bug_bucket/hqOvUgdGUbqiDyYs_low"

    at ChildProcess.exithandler (node:child_process:333:12)
    at ChildProcess.emit (node:events:376:20)
    at maybeClose (node:internal/child_process:1063:16)
    at Socket.<anonymous> (node:internal/child_process:449:11)
    at Socket.emit (node:events:376:20)
    at Pipe.<anonymous> (node:net:666:12)Error: Command failed: aws --endpoint-url http://localhost:8333 s3 cp /var/folders/r0/ktfq_wg90b18mrfb034n78wr0000gn/T/RmOVHeVYruzGXfBR_med s3://photagallery_medium_res
upload failed: ../../../../var/folders/r0/ktfq_wg90b18mrfb034n78wr0000gn/T/RmOVHeVYruzGXfBR_med to s3://photagallery_medium_res/RmOVHeVYruzGXfBR_med An error occurred (InternalError) when calling the PutObject operation (reached max retries: 2): We encountered an internal error, please try again.

    at ChildProcess.exithandler (node:child_process:333:12)
    at ChildProcess.emit (node:events:376:20)
    at maybeClose (node:internal/child_process:1063:16)
    at Socket.<anonymous> (node:internal/child_process:449:11)
    at Socket.emit (node:events:376:20)
    at Pipe.<anonymous> (node:net:666:12)Error: Command failed: aws --endpoint-url http://localhost:8333 s3 cp /var/folders/r0/ktfq_wg90b18mrfb034n78wr0000gn/T/eOnYrnljDijxetyR_med s3://photagallery_medium_res
upload failed: ../../../../var/folders/r0/ktfq_wg90b18mrfb034n78wr0000gn/T/eOnYrnljDijxetyR_med to s3://photagallery_medium_res/eOnYrnljDijxetyR_med An error occurred (InternalError) when calling the PutObject operation (reached max retries: 2): We encountered an internal error, please try again.

    at ChildProcess.exithandler (node:child_process:333:12)
    at ChildProcess.emit (node:events:376:20)
    at maybeClose (node:internal/child_process:1063:16)
    at Socket.<anonymous> (node:internal/child_process:449:11)
    at Socket.emit (node:events:376:20)
    at Pipe.<anonymous> (node:net:666:12)